packages/ffe-accordion-react/src/Accordion.md

```diff
@@ -29,7 +29,7 @@ til å begynne med (man kan fortsatt lukke det):
 const { WhiteAccordion, AccordionItem } = require('.');
 
 <WhiteAccordion>
-    <AccordionItem isOpen={true} title="Åpen!">
+    <AccordionItem defaultIsOpen={true} title="Åpen!">
```

Comment:
`defaultOpen` so it's in line with the implementation?

Reply:
argh, thanks!